agent/fw_wordpress.c

```diff
@@ -361,16 +379,27 @@ NR_PHP_WRAPPER(nr_wordpress_wrap_hook) {
    */
   NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_WORDPRESS);
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
+     && !defined OVERWRITE_ZEND_EXECUTE_DATA
+  if ((0 == NRINI(wordpress_hooks)) || (NULL == nr_stack_get_top(&NRPRG(wordpress_tags)))) {
```

Comment:
storing the result of `nr_stack_get_top(&NRPRG(wordpress_tags))` in a variable and referencing the variable instead could optimize 3 calls to `nr_stack_get_top()` down to 1 in this function.

Reply:
fixed in [de23c51](https://github.com/newrelic/newrelic-php-agent/pull/559/commits/de23c51e5e08e3661282d595b47809ec91d25ded)